modules/sovrnBidAdapter.js

```diff
@@ -158,7 +182,7 @@ export const spec = {
             dealId: sovrnBid.dealid || null,
             currency: 'USD',
             netRevenue: true,
-            mediaType: BANNER,
```

Comment:
Also for video we are going to not send the nurl, we will have it embedded in the creative.